odoo/addons/base/models/assetsbundle.py

```diff
@@ -722,6 +817,21 @@ def to_node(self):
                 ['data-asset-version', self.bundle.version],
             ]), self.with_header())
 
+    def with_header(self, content=None, minimal=True):
+        if minimal:
+            return super(JavascriptAsset, self).with_header(content)
+        else:
```

Comment:
User `super()` and dedent the `else` part:
```suggestion
        if minimal:
            return super().with_header(content)
        ...
```